flutter_local_notifications/example/lib/main.dart

```diff
@@ -1178,6 +1185,84 @@ class _HomePageState extends State<HomePage> {
           );
         });
   }
+
+  Future<void> _getActiveNotifications() async {
+    Widget activeNotificationsDialogContent =
+        await _getActiveNotificationsDialogContent();
+    await showDialog<void>(
+        context: context,
+        builder: (BuildContext context) {
+          return AlertDialog(
+            content: activeNotificationsDialogContent,
+            actions: [
+              FlatButton(
+                child: Text('OK'),
+                onPressed: () {
+                  Navigator.of(context).pop();
+                },
+              ),
+            ],
+          );
+        });
+  }
+
+  Future<Widget> _getActiveNotificationsDialogContent() async {
+    if (!Platform.isAndroid) {
+      return Text('"getActiveNotifications" is available only on Android');
+    }
+
+    DeviceInfoPlugin deviceInfo = DeviceInfoPlugin();
+    AndroidDeviceInfo androidInfo = await deviceInfo.androidInfo;
+    if (!(androidInfo.version.sdkInt >= 23)) {
+      return Text(
+        '"getActiveNotifications" is available only for Android API Level 23 and above',
+      );
+    }
+
+    try {
+      List<ActiveNotification> activeNotifications =
+          await flutterLocalNotificationsPlugin
+              .resolvePlatformSpecificImplementation<
+                  AndroidFlutterLocalNotificationsPlugin>()
+              ?.getActiveNotifications();
+
+      return Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Text(
+            'Active Notifications',
+            style: TextStyle(fontWeight: FontWeight.bold),
+          ),
+          Divider(color: Colors.black),
+          if (activeNotifications.isEmpty) Text('No active notification'),
+          if (activeNotifications.isNotEmpty)
+            ...activeNotifications
+                .map(
+                  (ActiveNotification aN) => Column(
```

Comment:
nit: Convention being used in the existing codebase is to stick with a single letter for variables used in an anonymous function/lambda expression. Having said that, should be able to use the `collection for` that is idiomatic to Dart and removes the need for spread, map and toList operations.

See this article for an example if you haven't come across it before https://medium.com/dartlang/announcing-dart-2-3-optimized-for-building-user-interfaces-e84919ca1dff

Reply:
I wasn't aware of `collection for` which I'll definitely use from now on! I've implemented it in this PR